[ERROR] File already exists: /raid/local/containers/enroot-data/user-1787/pytorch
Sliding Window Attention is enabled but not implemented for `sdpa`; unexpected results may be encountered.
Loading checkpoint shards:   0%|          | 0/37 [00:00<?, ?it/s]Loading checkpoint shards:   3%|▎         | 1/37 [00:01<00:46,  1.29s/it]Loading checkpoint shards:   5%|▌         | 2/37 [00:02<00:35,  1.02s/it]Loading checkpoint shards:   8%|▊         | 3/37 [00:02<00:29,  1.15it/s]Loading checkpoint shards:  11%|█         | 4/37 [00:03<00:26,  1.24it/s]Loading checkpoint shards:  14%|█▎        | 5/37 [00:04<00:24,  1.29it/s]Loading checkpoint shards:  16%|█▌        | 6/37 [00:04<00:23,  1.33it/s]Loading checkpoint shards:  19%|█▉        | 7/37 [00:05<00:23,  1.25it/s]Loading checkpoint shards:  22%|██▏       | 8/37 [00:06<00:21,  1.34it/s]Loading checkpoint shards:  24%|██▍       | 9/37 [00:07<00:19,  1.41it/s]Loading checkpoint shards:  27%|██▋       | 10/37 [00:07<00:18,  1.43it/s]Loading checkpoint shards:  30%|██▉       | 11/37 [00:08<00:17,  1.45it/s]Loading checkpoint shards:  32%|███▏      | 12/37 [00:09<00:17,  1.43it/s]Loading checkpoint shards:  35%|███▌      | 13/37 [00:09<00:17,  1.41it/s]Loading checkpoint shards:  38%|███▊      | 14/37 [00:10<00:15,  1.46it/s]Loading checkpoint shards:  41%|████      | 15/37 [00:11<00:14,  1.52it/s]Loading checkpoint shards:  43%|████▎     | 16/37 [00:11<00:13,  1.55it/s]Loading checkpoint shards:  46%|████▌     | 17/37 [00:12<00:12,  1.56it/s]Loading checkpoint shards:  49%|████▊     | 18/37 [00:13<00:12,  1.55it/s]Loading checkpoint shards:  51%|█████▏    | 19/37 [00:13<00:11,  1.57it/s]Loading checkpoint shards:  54%|█████▍    | 20/37 [00:14<00:10,  1.57it/s]Loading checkpoint shards:  57%|█████▋    | 21/37 [00:14<00:10,  1.57it/s]Loading checkpoint shards:  59%|█████▉    | 22/37 [00:15<00:10,  1.37it/s]Loading checkpoint shards:  62%|██████▏   | 23/37 [00:16<00:09,  1.42it/s]Loading checkpoint shards:  65%|██████▍   | 24/37 [00:17<00:08,  1.47it/s]Loading checkpoint shards:  68%|██████▊   | 25/37 [00:17<00:07,  1.51it/s]Loading checkpoint shards:  70%|███████   | 26/37 [00:18<00:07,  1.54it/s]Loading checkpoint shards:  73%|███████▎  | 27/37 [00:18<00:06,  1.58it/s]Loading checkpoint shards:  76%|███████▌  | 28/37 [00:19<00:05,  1.59it/s]Loading checkpoint shards:  78%|███████▊  | 29/37 [00:20<00:05,  1.59it/s]Loading checkpoint shards:  81%|████████  | 30/37 [00:20<00:04,  1.60it/s]Loading checkpoint shards:  84%|████████▍ | 31/37 [00:21<00:03,  1.62it/s]Loading checkpoint shards:  86%|████████▋ | 32/37 [00:22<00:03,  1.62it/s]Loading checkpoint shards:  89%|████████▉ | 33/37 [00:22<00:02,  1.61it/s]Loading checkpoint shards:  92%|█████████▏| 34/37 [00:23<00:01,  1.61it/s]Loading checkpoint shards:  95%|█████████▍| 35/37 [00:23<00:01,  1.63it/s]Loading checkpoint shards:  97%|█████████▋| 36/37 [00:24<00:00,  1.61it/s]Loading checkpoint shards: 100%|██████████| 37/37 [00:25<00:00,  1.67it/s]Loading checkpoint shards: 100%|██████████| 37/37 [00:25<00:00,  1.48it/s]
Traceback (most recent call last):
  File "/home/projects/fraudagent/hase-michael/V3/batch-large.py", line 78, in <module>
    main()
  File "/home/projects/fraudagent/hase-michael/V3/batch-large.py", line 74, in main
    df.to_csv(f"model_responses/{model_name.split('/')[-1]}/model_responses.csv", index=False)
  File "/usr/local/lib/python3.10/dist-packages/pandas/util/_decorators.py", line 333, in wrapper
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/pandas/core/generic.py", line 3967, in to_csv
    return DataFrameRenderer(formatter).to_csv(
  File "/usr/local/lib/python3.10/dist-packages/pandas/io/formats/format.py", line 1014, in to_csv
    csv_formatter.save()
  File "/usr/local/lib/python3.10/dist-packages/pandas/io/formats/csvs.py", line 251, in save
    with get_handle(
  File "/usr/local/lib/python3.10/dist-packages/pandas/io/common.py", line 749, in get_handle
    check_parent_directory(str(handle))
  File "/usr/local/lib/python3.10/dist-packages/pandas/io/common.py", line 616, in check_parent_directory
    raise OSError(rf"Cannot save file into a non-existent directory: '{parent}'")
OSError: Cannot save file into a non-existent directory: 'model_responses/job_20250528_2030_qwen_Qwen2.5-72B-Instruct_LoRA'
